src/App.js

```diff
@@ -1,106 +1,134 @@
-import React from 'react';
+import React, { useRef, useState } from 'react';
 import './App.css';
 
 import firebase from 'firebase/app';
 import 'firebase/firestore';
 import 'firebase/auth';
+import 'firebase/analytics';
```

Comment:
this is not currently being used, are you going to link it up to firebase?

Reply:
I think we might need add this in future